Core/GDCore/Project/Project.h

```diff
@@ -464,6 +465,13 @@ class GD_CORE_API Project : public ObjectsContainer {
   std::unique_ptr<gd::Object> CreateObject(const gd::String& type,
                                            const gd::String& name) const;
 
+  /**
+   * Create an object configuration of the given type with the specified name.
+   * 
+   * \param type The type of the object

```

Comment:
```suggestion
   * \param type The type of the object configuration.

```

Reply:
It's actually the type of the object of the configuration.